src/main/java/com/rakovets/course/java/core/practice/jcf_set/web_service/input_output_tools/LoginAndPasswordInputTool.java

```diff
@@ -0,0 +1,49 @@
+package com.rakovets.course.java.core.practice.jcf_set.web_service.input_output_tools;
+
+import com.rakovets.course.java.core.practice.jcf_set.web_service.UsersDataBase;
+import com.rakovets.course.java.core.practice.jcf_set.web_service.exceptions.LoginAlreadyExistsException;
+import com.rakovets.course.java.core.practice.jcf_set.web_service.exceptions.LoginDoesNotExistException;
+import java.util.Scanner;
+
+public class LoginAndPasswordInputTool {
+   Scanner scanner;
+   private UsersDataBase dataBase;
+
+   public LoginAndPasswordInputTool(UsersDataBase dataBase) {
+       scanner = new Scanner(System.in);
+       this.dataBase = dataBase;
+   }
+
+    public String chooseLogin()  {
+        System.out.println("Please input Login");
+        return scanner.nextLine();
+    }
+    public String choosePassword() {
+        System.out.println("Please choose Password");
+        return scanner.nextLine();
+    }
+
+    public String inputLogin() throws LoginDoesNotExistException {
+       System.out.println("Login:");
+       String login = scanner.nextLine();
+       if (!dataBase.getLoginAndPasswords().containsKey(login)){
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"